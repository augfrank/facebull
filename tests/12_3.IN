M0   C0   C1  2870
M1   C0   C3  2717
M2   C0   C9  2081
M3   C0   C11  2992
M4   C1   C6  2715
M5   C1   C7  2035
M6   C1   C9  2306
M7   C1   C10  2924
M8   C1   C11  2411
M9   C2   C0  2651
M10   C2   C1  2938
M11   C2   C5  2314
M12   C2   C6  2628
M13   C2   C8  2129
M14   C2   C10  2767
M15   C3   C1  2322
M16   C3   C4  2989
M17   C3   C7  2931
M18   C3   C10  2272
M19   C4   C1  2825
M20   C4   C2  2489
M21   C4   C3  2145
M22   C4   C6  2472
M23   C4   C8  2727
M24   C4   C11  2527
M25   C5   C4  2072
M26   C5   C9  2132
M27   C6   C2  2979
M28   C6   C4  2152
M29   C6   C5  2363
M30   C6   C8  2587
M31   C6   C11  2611
M32   C7   C3  2262
M33   C7   C4  2208
M34   C7   C5  2186
M35   C7   C6  2182
M36   C7   C10  2711
M37   C8   C2  2800
M38   C8   C3  2640
M39   C8   C9  2511
M40   C9   C0  2331
M41   C9   C1  2104
M42   C9   C4  2426
M43   C9   C7  2042
M44   C9   C10  2929
M45   C9   C11  2870
M46   C10   C0  2441
M47   C10   C3  2678
M48   C10   C5  2375
M49   C11   C2  2821
M50   C11   C3  2043
M51   C11   C5  2748
