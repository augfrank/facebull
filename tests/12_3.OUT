27458
2 5 11 18 24 25 30 35 37 41 46 50
